docs/dev/index.rst

```diff
@@ -1,6 +1,28 @@
 Contributing to ``stellarphot``
 -------------------------------
 
+Installation for one-time testing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you are testing a pre-release version of stellarphot we recommend setting up
+a virtual environment and installing stellarphot in this environment.
+
+Only use one of the methods below for making a virtual environment.
+
+Creating an environment with `conda` or `mamba` (use whichever one you have installed)::
+
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
+    pip install git+https://github.com/mwcraig/stellarphot.git@update-docs
+
+Creating an environment with `virtualenv`::
+
+
```

Comment:
You have two blank lines, I think you only need one.

Reply:
fixed